### PR TITLE
fix(pytest): Import borsh from near-sdk-rs directly

### PR DESCRIPTION
**Pull request by birchmd**
* `near-sdk-rs` now re-exports its `borsh`, so we do not need a separate dependency for it

### CODE REVIEW

File: pytest/empty-contract-rs/src/lib.rs [hunk 1]
```diff
@@ -1,5 +1,5 @@
-use borsh::{BorshDeserialize, BorshSerialize};
 use near_sdk::{env, metadata, near_bindgen};
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
```
Comment (bowenwang1996) [inline pytest/empty-contract-rs/src/lib.rs:2]: why is `self` needed?

Comment (birchmd) [reply pytest/empty-contract-rs/src/lib.rs:2]: It's used in some macros (e.g. `derive`).

File: pytest/empty-contract-rs/src/lib.rs [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 
 #[global_allocator]
 static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
```
Comment (evgenykuzyakov) [inline pytest/empty-contract-rs/src/lib.rs:5]:
```suggestion
static ALLOC: near_sdk::wee_alloc::WeeAlloc = near_sdk::wee_alloc::WeeAlloc::INIT;
```

File: pytest/empty-contract-rs/Cargo.toml [hunk 3]
```diff
@@ -12,8 +12,6 @@ serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 wee_alloc = "0.4.5"
```
Comment (evgenykuzyakov) [inline pytest/empty-contract-rs/Cargo.toml:13]: Also remove `serde` import above (can't comment on in with github UI)

```suggestion
```

Comment (birchmd) [reply pytest/empty-contract-rs/Cargo.toml]: Awesome, thanks. Done in https://github.com/nearprotocol/nearcore/pull/2991/commits/5c07c21f0057a0cf9a1c63810fcc1006daa1f8ac

File: pytest/empty-contract-rs/src/lib.rs (hunk 2 above)
Comment (birchmd) [reply pytest/empty-contract-rs/src/lib.rs]: Done in https://github.com/nearprotocol/nearcore/pull/2991/commits/5c07c21f0057a0cf9a1c63810fcc1006daa1f8ac

**Pull request merged**